routers/web/repo/render.go

```diff
@@ -44,20 +45,17 @@ func RenderFile(ctx *context.Context) {
 	isTextFile := st.IsText()
 
 	rd := charset.ToUTF8WithFallbackReader(io.MultiReader(bytes.NewReader(buf), dataRc), charset.ConvertOpts{})
+	ctx.Resp.Header().Add("Content-Security-Policy", "frame-src 'self'; sandbox allow-scripts")
```

Comment:
Should we also set [`frame-ancestors 'self'`](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Content-Security-Policy/frame-ancestors)?

Reply:
I think so. Maybe we can also update the original PR. Please send commits to this PR directly.

Reply:
I think we likely want to set only `frame-ancestors`, e.g. control where this page is embedded, not which pages it can embed.

Reply:
Assuming we want to backport this, I'd suggest doing the `frame-ancestors` change only on v1.23, so this is fine with me without.

Reply:
5b2d7b6a1fdc7795c5cc20e84cb9c2cc8265ad31

Reply:
My only concern about this is in case someone uses this renderer on external websites. Do we support this?

Reply:
OK. Let's revert it. 07e1ba9